<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="db" uuid="30e7159a-671a-4890-b4cc-49c66237540c">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:F:\Py1904\py1904-qiaochuanfa\demo1\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
    <data-source source="LOCAL" name="db [2]" uuid="06641042-c7c7-43c9-a44b-51ffa587aa1d">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:F:\Py1904\py1904-qiaochuanfa\demo2\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
    <data-source source="LOCAL" name="db [3]" uuid="d3f790ef-9689-4007-a9d1-473301a62015">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:F:\Py1904\py1904-qiaochuanfa\boke\db.sqlite3</jdbc-url>
      <driver-properties>
        <property name="enable_load_extension" value="true" />
      </driver-properties>
    </data-source>
  </component>
</project>